Project.toml

```diff
@@ -1,9 +1,11 @@
 name = "ObjectDetector"
 uuid = "3dfc1049-5314-49cf-8447-288dfd02f9fb"
 authors = ["Robert Luciani"]
-version = "0.3.3"
+version = "0.4.0"
```

Comment:
If you plan to do #105 after this, then it would be clearer to do the version bump in a third commit after these two.